tools/serverpod_cli/test/analyzer/entities/stateful_analyzer/protocol_validation/relation_one_to_many_test.dart

```diff
@@ -574,11 +574,11 @@ void main() {
           ListRelationDefinition);
     });
 
-    test('then the list relation is NOT linked with the companyId field', () {
+    test('then the list relation is NOT linked with the customerId field', () {
       var relation = classDefinition.findField('orders')?.relation
           as ListRelationDefinition;
 
-      expect(relation.foreignFieldName, isNot('companyId'));
+      expect(relation.foreignFieldName, isNot('customerId'));
```

Comment:
Would be good to reflect this change on line 547 as well. Right now the field is called company in the protocol file.